biocypher/_get.py

```diff
@@ -61,6 +63,25 @@ def __init__(
         self.is_dir = is_dir
 
 
+class APIRequest:
```

Comment:
Could this class not inherit from `Resource`? An `APIRequest` *is* a resource in the sense of our module, right? This would simplify type hinting, and we would only have to check which class is passed when deciding between pooch and requests.

Reply:
We could even make two distinct classes, `FileDownload` and `APIRequest`, which both inherit from an abstract base class `Resource`, to make the distinction even more clear. Thoughts about that?

Reply:
@slobentanzer yes, you are right. I will make two classes inherit from abstract `Resource`：）